game/static/game/css/opening_trainer.css

```
@@ -274,3 +274,53 @@ body{
+
+#get-hint-btn:disabled {
+    background: #444;
+    color: #888;
+    border-color: #444;
+    cursor: not-allowed;
+}
+
+/* Pulse effect for the start (from) square of the hint */
+@keyframes hint-pulse {
+    0% {
+        background-color: rgba(240, 192, 64, 0.4) !important;
```

Stylelint (17.14.0)

[error] Disallowed !important (keyframe-declaration-no-important)

(keyframe-declaration-no-important)